<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetWidget</class>
 <widget class="QWidget" name="SetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置空调参数</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="Modelabel">
         <property name="text">
          <string>模式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ModecomboBox">
         <item>
          <property name="text">
           <string>制冷</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>制热</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="DefaultFanlabel">
         <property name="text">
          <string>默认风速</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="DefaultFanBox">
         <item>
          <property name="text">
           <string>低</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>中</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>高</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="DefaultTemplabel">
         <property name="text">
          <string>默认温度</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DefaultTempEdit">
         <property name="text">
          <string>22</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="TempLowlabel">
       <property name="text">
        <string>最低温度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLineEdit" name="TempLowEdit">
       <property name="text">
        <string>18</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="TempHighlabel">
       <property name="text">
        <string>最高温度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLineEdit" name="TempHighEdit">
       <property name="text">
        <string>25</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="FeeLlabel">
       <property name="text">
        <string>费    率</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="FeeLEdit">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>元/度</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" rowspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>风速</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string> 低</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="FanLEdit">
       <property name="text">
        <string>0.33</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>度/分钟</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string> 中</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="FanMEdit">
       <property name="text">
        <string>0.5</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>度/分钟</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string> 高</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="FanHEdit">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>度/分钟</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="SetConfirmButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
